### DISCUSSION TITLE
SWR + Material UI (Collapse)

### DISCUSSION THREAD

**Discussion by della-monica** [vercel/swr#564] [General]
I'm using swr with material UI. 
![screenshot-sgi-local geocapital com br_8080-2020 07 28-12_16_36](https://user-images.githubusercontent.com/68914188/88685681-93cfdb00-d0cc-11ea-96c6-2d286a2469c6.png)

When the data reloads, the component (collapse) that was open before now is closed.

Is there any way to avoid this?

**Comment by sergiodxa** [comment]
Looks like a MUI issue, your component is remounting and it’s not keeping the open/close state
